Assets/Scenes/20-MainWorld/Scripts/LoadingSceneSetup.cs

```diff
@@ -54,6 +57,7 @@ public void Load()
     void OnSceneLoaded(Scene scene, LoadSceneMode loadSceneMode)
     {
         GameObject gamemode = GameObject.FindGameObjectWithTag("GamemodeSelect");
+        Debug.Log("load scene id" + sceneID);
```

Comment:
debug log?

Reply:
ill remove it but it aint mine, ill take flame if i do get it